client/components/Header/style.css

```diff
@@ -0,0 +1,20 @@
+.header {
+	display: flex;
+	margin-bottom: 3em;
+}
+
+a {
+	padding: 35px 25px;
+	margin: -6px 6px 0;
+	border-radius: 6px;
+	font-size: 16px;
+	color: #b1b1b1;
+	border: 1px solid #dadada;
```

Comment:
colors and various sizes are best kept in a file where we would store global variables for reuse